### ISSUE TITLE
For Angular Core it prints "The variable `HttpParams` should be named `HttpParameters`"

### ISSUE THREAD

**Issue by muuvmuuv** [sindresorhus/eslint-plugin-unicorn#1251]
I am using Angular and since the latest update it reports internal lib imports as abbreviation error which results in import errors.

```
The variable `HttpParams` should be named `HttpParameters`. A more descriptive name will do too.eslintunicorn/prevent-abbreviations
```

```js
import { HttpHeaders, HttpParams } from '@angular/common/http'
```

**Comment by sindresorhus** [comment]
```js
import { HttpHeaders, HttpParams as HttpParameters } from '@angular/common/http'
```

**Comment by sindresorhus** [comment]
Yeah, that's a bug.

**Comment by fisker** [comment]
I can't reproduce, and we have this case tested https://github.com/sindresorhus/eslint-plugin-unicorn/blob/4bf09c4620d60fe3f0749352f85357c296eab0d7/test/prevent-abbreviations.mjs#L1349

What parser are you using? And what options do you use? Escpecially [`checkShorthandImports`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prevent-abbreviations.md#checkshorthandimports)

**Comment by fisker** [comment]
It's bug, it fails on `babel` and `typescript` parser.
